pkg/visor/init.go

```diff
@@ -766,3 +819,40 @@ func getErrors(ctx context.Context) chan error {
 	}
 	return errs
 }
+
+func stunDetails(stunServers []string, log *logging.Logger) *snet.StunClient {
```

Comment:
Can we move it to a separate package? Perhaps into snet itself, since it's used by it.

Reply:
Yes I'll move the stun code to snet.